### DISCUSSION TITLE
Biomarkers in README.md Step3

### DISCUSSION THREAD

**Discussion by ApolloLSY** [dayoonkwon/BioAge#18] [Q&A] [answered]
Hi there,

Thanks for creating this package! 
I was wondering why you use more biomarkers in README.md Step3 than you do in the phenoage_calc function body?
![c29030639643fb7b940b99906d56aca](https://github.com/dayoonkwon/BioAge/assets/78656349/d6d9e69d-393d-4ab6-94a4-cb81b9162c0e)
![c7313aa5c3f489f82da02b1f73eecff](https://github.com/dayoonkwon/BioAge/assets/78656349/61c40057-4c52-4afd-b5ad-8f51ae07deeb)
For example, I found that "lncreat_umol" and "uap" are not used in function phenoage_calc.
![9a3283f94b9de8fbc4470261838bff6](https://github.com/dayoonkwon/BioAge/assets/78656349/6fe56436-7599-4e4d-8862-0c465837479c)

Could you please explain why these parameters are different? Thanks again!

**Comment by dayoonkwon** [comment, marked as answer]
The one in the bracket is the [original phenoage value](https://www.aging-us.com/article/101414/text), which is saved in `phenoage0`.
